### PR TITLE
Lower resource requests to idle usage

### PR DESCRIPTION
**Pull request by pintohutch**
Lower resource requests to match idle usage

### CODE REVIEW

File: cmd/operator/deploy/operator/11-rule-evaluator.yaml [hunk 1]
```diff
@@ -103,8 +103,8 @@ spec:
           limits:
             memory: 1G
           requests:
-            cpu: 8m
-            memory: 32M
+            cpu: 1m
+            memory: 16M
```
Comment (TheSpiritXIII) [inline cmd/operator/deploy/operator/11-rule-evaluator.yaml:107]: Should we maybe keep this consistent with `cmd/operator/deploy/rule-evaluator/01-deployment.yaml`?

File: cmd/operator/deploy/operator/12-alertmanager.yaml [hunk 2]
```diff
@@ -82,8 +82,8 @@ spec:
           limits:
             memory: 128M
           requests:
-            cpu: 8m
-            memory: 32M
+            cpu: 1m
+            memory: 16M
```
Comment (TheSpiritXIII) [inline cmd/operator/deploy/operator/12-alertmanager.yaml:86]: Should we maybe keep this consistent with `examples/alertmanager.yaml`?

**Review by TheSpiritXIII — APPROVED** [review]
LGTM! Just added some minor optional nits!

File: cmd/operator/deploy/operator/12-alertmanager.yaml (hunk 2 above)
Comment (pintohutch) [reply cmd/operator/deploy/operator/12-alertmanager.yaml:86]: Thx - done!

File: cmd/operator/deploy/operator/11-rule-evaluator.yaml (hunk 1 above)
Comment (pintohutch) [reply cmd/operator/deploy/operator/11-rule-evaluator.yaml:107]: Thx - done!

**Pull request merged**